doc/dev/index.rst

```diff
@@ -179,6 +179,100 @@ with::
 
 For more ``vstart.sh`` examples, see :doc:`/dev/quick_guide`.
 
+What is merged where and when ?
+===============================
+
+Commits are merged into branches according to criterions that change
+during the lifecycle of a Ceph release. This chapter is the inventory
+of what can be merged in which branch at a given point in time.
+
+Development releases (i.e. x.0.z)
+---------------------------------
+
+What ?
+^^^^^^
+
+ * features and bug fixes
+
+Where ?
+^^^^^^^
+
+Features are merged to the master branch. Bug fixes should be (but it is not mandatory)
+merged in the branch of the stable release (hammer for 0.94.x, infernalis for
+9.2.x, etc.).
+
+When ?
+^^^^^^
+
+After the stable release candidates of the previous release starts phase 2 (see below).
```

Comment:
For example: the "jewel" named branch was created when the infernalis release candidates entered phase 2. From this point on, master was no longer associated with infernalis. As soon as the named branch of the next stable release is created, master starts getting periodically merged into it.